internal/pkg/aws/ec2/ec2.go

```diff
@@ -83,8 +84,34 @@ func New(s *session.Session) *EC2 {
 	}
 }
 
-// ListVPC returns IDs of all VPCs.
-func (c *EC2) ListVPC() ([]string, error) {
+// VPC contains the ID and name of a VPC.
+type VPC struct {
+	ID   string
+	Name string
+}
+
+// String formats the elements of a VPC into a display-ready string.
+func (v *VPC) String() (string, error) {
```

Comment:
Do we need the error? it's always nil and this way it would satisfy the [fmt.Stringer](https://golang.org/pkg/fmt/#Stringer) interface